comment/templates/comment/dislike_icon.html

```diff
@@ -0,0 +1,6 @@
+{% load comment_tags %}
+{% block dislike_img_icon %}
+<svg class="comment-reaction-icon reaction-like" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill={% if comment|add_one_arg:user|has_reacted:"like" %}"#ffc966"{% else %}"none"{% endif %} stroke="#427297" stroke-width="1.7" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
```jinja2
{% if comment|add_one_arg:user|has_reacted:"like" %}
```
We agreed to change the template tags!

Reply:
https://github.com/Radi85/Comment/pull/10#discussion_r425442592
I had updated my comment after I saw we couldn't use the simple tag with if tag.

Reply:
Tested:
```jinja2
{% has_reacted comment user "like" as user_has_reacted %}

{% if user_has_reacted %}"#ffc966"{% else %}"none"{% endif %}
```
I am not happy with filter chain. So far everything looks good!

Reply:
okay, will check and revert to you.